### ISSUE TITLE
Revise pack editing for large packs

### ISSUE THREAD

**Issue by ajaysmurthy** [MTG/freesound#596]
I am working with the pack http://www.freesound.org/people/ajaysm/packs/14056

I get a 404 Not Found Error whenever I edit the pack description and submit the edits. The pack in question has 236 sounds and was uploaded to FS using a script. 

It might be a django timeout.

**Comment by alastair** [comment]
We still need pagination. I'm not sure if we completely fixed the saving too - I believe sound saving/pack generation can still be made faster

**Comment by ffont** [comment]
After some more implementation work and more discussion we concluded there are a number of things which are not right in the pack edit page:

- There is no pagination for the current sounds of the pack nor for the available user sounds
- The code for saving the pack is a bit messy and not optimised: https://github.com/MTG/freesound/blob/4e4b91fa45adabb5294487eeed384596425fd8bb/sounds/forms.py#L154-L182

The first point is mostly for UI purposes, but simple implementations of a paginator won't work as we need to be able to keep on adding/removing sound ids to a js variable while switching pages. UI-wise, the best would probably to do an ajax paginator here. Nevertheless this is not a commonly used feature, and for small packs the current UI will be ok. Considering that we're updating the whole interface soon then we decided there is no need to work on the pagination issues now.

Regarding the second point it is still worth looking at it, but it is not main priority.
